### DISCUSSION TITLE
Bad root server path for configuration file using Docker

### DISCUSSION THREAD

**Discussion by peacefulotter** [playframework/playframework#11793] [General]
### Play Version

Play Version 2.8.19

### API

Scala: 2.13.10
Java 1.8.0_362

### Operating System

Linux echo 5.19.0-40-generic #41~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Mar 31 16:00:14 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

### JDK

openjdk version "18" 2022-03-22
OpenJDK Runtime Environment (build 18+36-2087)
OpenJDK 64-Bit Server VM (build 18+36-2087, mixed mode, sharing)

### Library Dependencies

  "com.typesafe.play" %% "play-slick" % "5.1.0",
  "org.postgresql" % "postgresql" % "42.5.4"

### Expected Behavior

1. Build docker container using `sbt "Docker \ publishLocal"`
2. Run container using `docker run --rm -p 9000:9000 playground-api`
3. Have the container running without error

### Actual Behavior

When entering the docker run command, a Bad Server Root path exception is thrown.
```
/opt/docker/bin/play-scala-seed: line 99: is_cygwin: not found
Bad root server path: /opt/docker/-Dconfig.resource=/opt/docker/conf/production.conf
```

### Reproducible Test Case

`/conf/production.conf`
```
include "application.conf"
application.mode=prod
play.http.secret.key="SECRET GENERATED USING sbt playGenerateSecret"
%production%.application.baseUrl="https://subdomain.example.com"
play.filters.hosts {
  allowed = [".example.com", "localhost:9000"]
}
```

`build.sbt`
```
...
Docker / version := sys.env.getOrElse("BUILD_NUMBER", "0")
Docker / daemonUserUid  := None
Docker / daemonUser := "daemon"
dockerExposedPorts := Seq(9000)
dockerBaseImage := "openjdk:8-jre-alpine"
dockerUpdateLatest := true
dockerEntrypoint := Seq("/opt/docker/bin/play-scala-seed", "-Dconfig.resource=/opt/docker/conf/production.conf")
```

Running `sbt "Docker \ publishLocal"` produces the following Dockerfile:
```
FROM openjdk:8-jre-alpine
LABEL MAINTAINER="name@example.com
WORKDIR /opt/docker
COPY --chown=daemon:root opt /opt
EXPOSE 9000
USER daemon
ENTRYPOINT ["/opt/docker/bin/play-scala-seed", "-Dconfig.resource=/opt/docker/conf/production.conf"]
CMD []
```

### What I have tried
1. Changing the dockerEntrypoint from the build.sbt file to "-Dconfig.resource=/conf/production.conf" or "-Dconfig.resource=production.conf"
2. Putting the -Dconfig.resource to the docker run command: docker run ... playground-api -Dconfig.resource=./conf/production.conf

**Comment by mkurz** [comment]
```
/opt/docker/bin/play-scala-seed: line 99: is_cygwin: not found
```
This seems strange to me. The script generated by sbt-native package should contain a function something like:
```sh
# Uses uname to detect if we're in the odd cygwin environment.
is_cygwin() {
  local os=$(uname -s)
  case "$os" in
    CYGWIN*) return 0 ;;
    *)  return 1 ;;
  esac
}
```
This is what it generates me when running `sbt stage`

**Reply by peacefulotter** [reply]
As per [this](https://dvirf1.github.io/play-tutorial/posts/dockerize-the-app/) link, one can ignore this error message:
"Run a docker container of your application: docker run --rm -p 9000:9000 playground-api.
If you see a message like is_cygwin: not found you can ignore it. It is an output of the start script"

**Comment by peacefulotter** [comment]
Hi, I am curious if this has been answered somewhere else or fixed. Any news?
